mmcls/models/backbones/efficientnet_v2.py

```diff
@@ -0,0 +1,395 @@
+import math
+from functools import partial
+from typing import Callable, List, Optional, Tuple
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from mmcv.cnn.bricks import DropPath
+from mmengine.model import Sequential
+from torch import Tensor
+
+from mmcls.models.backbones.base_backbone import BaseBackbone
+from mmcls.registry import MODELS
+
+
+# Copy from timm
+# Calculate asymmetric TensorFlow-like 'SAME' padding for a convolution
+def get_same_padding(x: int, k: int, s: int, d: int):
+    return max((math.ceil(x / s) - 1) * s + (k - 1) * d + 1 - x, 0)
+
+
+# Dynamically pad input x with 'SAME' padding for conv with specified args
+def pad_same(x,
+             k: List[int],
+             s: List[int],
+             d: List[int] = (1, 1),
+             value: float = 0):
+    ih, iw = x.size()[-2:]
+    pad_h, pad_w = get_same_padding(ih, k[0], s[0], d[0]), get_same_padding(
+        iw, k[1], s[1], d[1])
+    if pad_h > 0 or pad_w > 0:
+        x = F.pad(
+            x,
+            [pad_w // 2, pad_w - pad_w // 2, pad_h // 2, pad_h - pad_h // 2],
+            value=value)
+    return x
+
+
+def conv2d_same(x,
+                weight: torch.Tensor,
+                bias: Optional[torch.Tensor] = None,
+                stride: Tuple[int, int] = (1, 1),
+                padding: Tuple[int, int] = (0, 0),
+                dilation: Tuple[int, int] = (1, 1),
+                groups: int = 1):
+    x = pad_same(x, weight.shape[-2:], stride, dilation)
+    return F.conv2d(x, weight, bias, stride, (0, 0), dilation, groups)
+
+
+class Conv2dSame(nn.Conv2d):
+    """Tensorflow like 'SAME' convolution wrapper for 2D convolutions."""
+
+    def __init__(self,
+                 in_channels,
+                 out_channels,
+                 kernel_size,
+                 stride=1,
+                 padding=0,
+                 dilation=1,
+                 groups=1,
+                 bias=True):
+        super(Conv2dSame,
+              self).__init__(in_channels, out_channels, kernel_size, stride, 0,
+                             dilation, groups, bias)
+
+    def forward(self, x):
+        return conv2d_same(x, self.weight, self.bias, self.stride,
+                           self.padding, self.dilation, self.groups)
+
+
+class ConvBNAct(nn.Module):
+
+    def __init__(self,
+                 in_planes: int,
+                 out_planes: int,
+                 kernel_size: int = 3,
+                 stride: int = 1,
+                 groups: int = 1,
+                 padding: int = 1,
+                 norm_layer: Optional[Callable[..., nn.Module]] = None,
+                 activation_layer: Optional[Callable[..., nn.Module]] = None):
+        super(ConvBNAct, self).__init__()
+
+        padding = (kernel_size - 1) // 2
+        if norm_layer is None:
+            norm_layer = nn.BatchNorm2d
+        if activation_layer is None:
+            activation_layer = nn.SiLU  # alias Swish
```

Comment:
I have read the [silu](https://pytorch.org/docs/stable/generated/torch.nn.SiLU.html) in torch doc and [mmcv.swish](https://github.com/open-mmlab/mmcv/blob/fb39e1e568c49fcd97f9a56cb70a89d6693738c7/mmcv/cnn/bricks/swish.py#L8). I think here can use the `mmcv.swish`.

Reply:
if swith is same as the silu ,so i can use the ConvModule, I will update the main model file

Reply:
yes, i will change all my code to mmcls style, not difficult but need some time